packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -56,6 +56,11 @@ const propTypes = {
    */
   rowHeaderIndex: PropTypes.number,
   /**
+   * Function that is called when a resizable column is resized. Parameters: `onRequestColumnResize(columnId, requestedWidth)`
```

Comment:
For consistency with the rest of the prop docs:
```suggestion
   * Function that is called when a resizable column is resized. Parameters: `function(columnId, requestedWidth)`
```